### ISSUE TITLE
Cannot use --parameter-overrides when invoking local lambda

### ISSUE THREAD

**Issue by celowsky** [aws/aws-toolkit-jetbrains#1542]
**Describe the bug**
When trying to locally invoke an AWS lambda function with a run configuration that has `--parameter-overrides` defined in **Local Invoke Args**,  the invocation will fail, outputting a message like the following:

```bash
/home/linuxbrew/.linuxbrew/bin/sam build --template /path/to/project/template.yaml --build-dir /path/to/project/.aws-sam/build --parameter-overrides LLAYER=MyLambda:Arn MyLambdaFunction
Usage: sam build [OPTIONS] [FUNCTION_IDENTIFIER]
Try "sam build --help" for help.

Error: Invalid value for "--parameter-overrides":  MyLambdaFunction is not in valid format. It must look something like 'ParameterKey=KeyPairName,ParameterValue=MyKey ParameterKey=InstanceType,ParameterValue=t1.micro' or 'KeyPairName=MyKey InstanceType=t1.micro'
```

**To reproduce**

1. Create a sam template.yaml with one parameter and one lambda function
1. Create a run configuration to locally invoke the lambda
1. Edit the configuration under the **SAM CLI** section to include a parameter override like `--parameter-overrides LLAYER=arn:aws:lambda:us-east-2:018447807893:layer:chromelayer:18`
1. Run the configuration. See the valid format error appear.

**Expected behavior**
The command generated by Intellij adds the `--parameter-overrides` and its arguments after the lambda function name to prevent this error from displaying.

**Your Environment**
- OS: Linux Mint 19.2
- JetBrains product: IntelliJ Ultimate Edition
- JetBrains product version: 2019.3.3
- AWS Toolkit version: 1.10-193
- SAM CLI version: 0.40.0
- JVM/Python version: python 3.7.5

**Comment by abrooksv** [comment]
I was not able to reproduce this:
![Screen Shot 2020-02-18 at 9 49 44 AM](https://user-images.githubusercontent.com/8992246/74763173-2cb41d80-5234-11ea-87a4-7145ccd02d7c.png)

Build Logs:
```
/usr/local/bin/sam build --template IdeaProjects/untitled31/template.yaml --build-dir IdeaProjects/untitled31/HelloWorldFunction/.aws-sam/build HelloWorldFunction
Building resource 'HelloWorldFunction'
```

Invoke Logs:
```
/usr/local/bin/sam local invoke --template IdeaProjects/untitled31/HelloWorldFunction/.aws-sam/build/template.yaml --event "/private/var/folders/q5/cfv4k63519vcllnnpwz5zsj0wnjvck/T/[Local] HelloWorldFunction-event1.json" HelloWorldFunction --parameter-overrides LLAYER=MyLambda:Arn
Invoking helloworld.App::handleRequest (java8)
```

**Comment by abrooksv** [comment]
Oh, for build it seems bugged. It is not handling it correctly like you said. I will cut a issue to SAM team since the function identifier is supposed to be last according to docs

**Comment by abrooksv** [comment]
Cut issue: https://github.com/awslabs/aws-sam-cli/issues/1799

**Comment by celowsky** [comment]
My mistake, yes it seems to only affect build. Thanks for cutting the issue 👍

**Comment by abrooksv** [comment]
Fix has been merged into master

**Comment by abrooksv** [comment]
Fix has been released in 1.11
